web-client/integration-tests/sentWorkItemsExpireAfterNDays.test.ts

```diff
@@ -60,6 +61,21 @@ describe('verify old sent work items do not show up in the outbox', () => {
     workItemIdN = applicationContext.getUniqueId();
     workItemIdNPlus1 = applicationContext.getUniqueId();
 
+    const docketEntry: RawDocketEntry = {
```

Comment:
Before, we relied on the nested docket entry saved as part of work item: we did not have to have a "real" docket entry saved into the db. Now, because our query must join `dwWorkItem` with `dwDocketEntry` to get the docket entry data, we need a "real" docket entry saved in the db. The same goes for the test `servedWorkItemsExpireAfter7Days`.